nullaway/src/main/java/com/uber/nullaway/handlers/InferredJARModelsHandler.java

```diff
@@ -66,6 +69,19 @@ public InferredJARModelsHandler() {
     super();
     argAnnotCache = new LinkedHashMap<>();
     loadedJars = new HashSet<>();
+    // Load Android models
+    try {
+      InputStream androidStubxIS =
+          Class.forName(ANDROID_MODEL_CLASS)
+              .getClassLoader()
+              .getResourceAsStream(ANDROID_ASTUBX_LOCATION);
+      if (androidStubxIS != null) {
+        parseStubStream(androidStubxIS, "android.jar: " + ANDROID_ASTUBX_LOCATION);
+        LOG(DEBUG, "DEBUG", "Loaded Android RT models.");
+      }
+    } catch (Exception e) {
```

Comment:
I'd still distinguish `ClassNotFoundException` as that is an expected exception rather than a generic error. Just log a message there in `DEBUG` level and add a comment that we expect it to be raised if the Android JarInfer Models jar is missing.

Also, I keep adding "JarInfer" when I talk about this models, because we want to distinguish them from the `LibraryModels` subclasses. Not saying you should change the constant names to include that, but make sure you make it clear in the comments that this are the JarInfer models.